From 7de67f717a53b56864a5c8be2bbd07aacbc4c807 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Mon, 16 Jun 2025 07:51:58 -0700
Subject: [PATCH] add ignore bucket

---
 download_from_google_storage.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/download_from_google_storage.py b/download_from_google_storage.py
index 58d4913cb..446aa3cf0 100755
--- a/download_from_google_storage.py
+++ b/download_from_google_storage.py
@@ -550,6 +550,9 @@ def main(args):
 
   (options, args) = parser.parse_args()
 
+  if options.bucket.includes('chromium-tools-traffic_annotation'):
+    retrun 0
+
   # Make sure we should run at all based on platform matching.
   if options.platform:
     if options.auto_platform:
-- 
2.49.0.windows.1

